engine/src/main/scala/cromwell/engine/workflow/lifecycle/WorkflowExecutionActor.scala

```diff
@@ -182,6 +188,7 @@ final case class WorkflowExecutionActor(workflowId: WorkflowId, workflowDescript
 
   when(WorkflowExecutionInProgressState) {
     case Event(BackendJobExecutionSucceededResponse(jobKey, callOutputs), stateData) =>
+      sender() ! PoisonPill
```

Comment:
In theory the sub-actors are already killing themselves so this should be redundant. But that doesn't necessarily mean its a bad idea!